Makefile

```diff
@@ -122,10 +123,16 @@ dependencies: bin/grpcurl
 	  fi \
 	done;
 
+tmp_dir=$(shell mktemp -d)
 bin/grpcurl:
```

Comment:
Will this lack of dependency causing `bin/grpcurl` to be rebuilt every time `make dev` is run? I knew it did that for the old `curl` downloaded version and I had to comment it out every time to speed up development.

Reply:
I won't - notice the target name is the file name.

Reply:
Tested locally and confirmed :)

Reply:
Thanks @dndx!